test/e2e-tests/tests/pipelines_test.go

```diff
@@ -81,74 +80,125 @@ func init() {
-		if err != nil {
-			t.Fatal(err.Error())
-		}
+	pipelineRun, err := support.ReadFileAsPipelineRun(AIEdgeE2ETensorflowHousingPipelineRunRelativePath)
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	if config.GitFetchConfig.SelfSignedCert != "" {
+		support.MountConfigMapAsWorkspaceToPipelineRun("git-self-signed-cert", "git-ssl-cert", &pipelineRun)
+	}
+
+	support.SetPipelineRunParam("target-image-tag-references", support.NewArrayParamValue(config.TargetImageTags), &pipelineRun)
+	support.SetPipelineRunParam("git-containerfile-repo", support.NewStringParamValue(config.GitFetchConfig.ContainerFileRepo), &pipelineRun)
+	support.SetPipelineRunParam("git-containerfile-revision", support.NewStringParamValue(config.GitFetchConfig.ContainerFileRevision), &pipelineRun)
+	support.SetPipelineRunParam("containerRelativePath", support.NewStringParamValue(config.GitFetchConfig.ContainerRelativePath), &pipelineRun)
+	support.SetPipelineRunParam("git-model-repo", support.NewStringParamValue(config.GitFetchConfig.ModelRepo), &pipelineRun)
+	support.SetPipelineRunParam("modelRelativePath", support.NewStringParamValue(config.GitFetchConfig.ModelRelativePath), &pipelineRun)
+	support.SetPipelineRunParam("git-model-revision", support.NewStringParamValue(config.GitFetchConfig.ModelRevision), &pipelineRun)
+	support.SetPipelineRunParam("model-dir", support.NewStringParamValue(config.GitFetchConfig.ModelDir), &pipelineRun)
+
+	_, err = config.Clients.PipelineRun.Create(ctx, &pipelineRun, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err.Error())
 	}
 
-	err = WaitForAllPipelineRunsToComplete(ctx, &clients)
+	err = WaitForAllPipelineRunsToComplete(ctx, config)
 	if err != nil {
 		t.Fatal(err.Error())
 	}
 }
 
-func Test_GitOpsPipelineRunsComplete(t *testing.T) {
+func Test_GitOpsUpdatePipeline(t *testing.T) {
 	ctx := CreateContext()
 
-	options, err := support.GetOptions()
+	config, err := support.GetConfig()
 	if err != nil {
-		t.Fatal(err.Error())
+		panic(err.Error())
 	}
 
-	clients, err := support.CreateClients(options.ClusterNamespace)
+	if !config.GitOpsConfig.Enabled {
+		t.Skipf("skipping %v, Gitops pipeline not configured", t.Name())
+	}
+
+	if !config.GitFetchConfig.Enabled && !config.S3FetchConfig.Enabled {
+		t.Skipf("skipping %v, neither fetch methods were not configured", t.Name())
+	}
+
+	clients, err := support.CreateClients(config.Namespace)
 	if err != nil {
 		t.Fatal(err.Error())
 	}
 
-	pipelineRunPaths := []string{GitOpsUpdateTensorflowHousingPipelineRunRelativePath, GitOpsUpdateBikeRentalsPipelineRunRelativePath}
+	// the pipeline runs we can run here are based on the fetch models used by the mlops pipeline
+	// because this pipeline depends on the finished pipeline run of the mlops pipeline
+	// -> https://issues.redhat.com/browse/RHOAIENG-4982
+	// there is also a possibilty to split these based on the fetch types like the mlops pipeline
+	var pipelineRunPaths []string
```

Comment:
But to get the `buildah-sha` do we not need to run the pipeline before to get it out of the results?

Reply:
Yes, to get the "fresh" image sha, we need to run it.

Reply:
Okay I think I understand now, we need to get the results from the mlops pipeline to get the value for `image-digest`. Even though now the gitops pipeline doesn't depend on it directly for these tests they still do just that now we need to do the magic of getting the sha from the first tests.

The comment
```go
// the pipeline runs we can run here are based on the fetch models used by the mlops pipeline
// because this pipeline depends on the pipelines results as it needs the `image-digest` value
```